ddtrace/settings/integration.py

```diff
@@ -152,3 +152,8 @@ def __repr__(self):
         cls = self.__class__
         keys = ", ".join(self.keys())
         return "{}.{}({})".format(cls.__module__, cls.__name__, keys)
+
+    def copy(self):
```

Comment:
is this used anywhere?

Reply:
Yes, so when a clone a Pin via `Pin.clone()`, the Pin's Integration Config object was being copied via `dict.copy()`, resulting in a dict result instead of IntegrationConfig object. This resulted in errors during the new django tests since we clone Pins during test setup. @brettlangdon FYI about this change and lmk thoughts.